app/pdfs/donation_pdf.rb

```diff
@@ -20,9 +20,13 @@ def initialize(donation)
         @address = nil
         @email = nil
       when Donation::SOURCES[:product_drive]
-        @name = donation.product_drive_participant.business_name
-        @address = donation.product_drive_participant.address
-        @email = donation.product_drive_participant.email
+        if donation.product_drive_participant
+          @name = donation.product_drive_participant.business_name
+          @address = donation.product_drive_participant.address
+          @email = donation.product_drive_participant.email
+        else
+          @name = donation.product_drive.name
```

Comment:
I think you are missing this requirement in your fix:

> If it's a product drive, but without a product drive participant, put "Product Drive -- [the name of the product drive]" there instead.

If you just call `name` on the product drive, that prefix "Product Drive" will be missing.

Reply:
Updated! [Added label for product drive](https://github.com/rubyforgood/human-essentials/pull/4831/commits/0257ea917652c5f664ecd06e3b653d1916c739a4)

Reply:
[Added Product Drive Label.pdf](https://github.com/user-attachments/files/18049158/Added.Product.Drive.Label.pdf)